pkg/server/server.go

```diff
@@ -684,6 +695,12 @@ func (s *Server) WriteAuthorizationModel(ctx context.Context, req *openfgav1.Wri
 		}
 	}
 
+	if slices.Contains(s.experimentals, ExperimentalRejectConditions) {
+		if req.Conditions != nil {
```

Comment:
What if a type restriction references a condition but the condition is undefined? What should be the server response behavior there? Should we return the validation error because we reference a condition that is undefined or should we prioritize the `codes.Unimplemented` status code due to conditions not being supported?

Personally, if you've disabled conditions I'm leaning toward the default favoring `codes.Unimplemented`. If we go this route will need a little more validation checking here on the type restrictions.

Reply:
took a stab at this in [658aafc](https://github.com/openfga/openfga/pull/1169/commits/658aafcf78aa410421834a9ac4577b401aa72ec4)